.github/workflows/ci-build.yaml

```diff
@@ -1,9 +1,8 @@
 name: CI
 
 # We test all supported Python versions as follows:
-# - 3.10 : Documentation build
-# - 3.10 : Part of Matrix with NumPy dispatch
-# - 3.10 : Part of Matrix
+# - 3.11 : Documentation build
+# - 3.11 : Part of Matrix with NumPy dispatch
 # - 3.11 : Part of Matrix
```

Comment:
We could probably just remove these comments altogether – they look to be a couple years out-of-date